extensions/quick-access/src/search-pinned-directories.tsx

```diff
@@ -123,6 +125,24 @@ export default function Command() {
                               }}
                             />
                           </ActionPanel.Section>
+                          <ActionPanel.Section title="Open With...">
```

Comment:
Any particular reason why not using the [OpenWith action](https://developers.raycast.com/api-reference/user-interface/actions#action.openwith)?

Reply:
Thanks for the reminder! The ones that are not used because I didn't notice such a useful component before.
I've submitted a commit to make the changes。

Another small problem, using <Action.OpenWith/>, the console keeps warning, am I using the wrong method?

```
 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    at Lc (/Applications/Raycast.app/Contents/Resources/RaycastCommands_RaycastCommands.bundle/Contents/Resources/api/node_modules/@raycast/api/index.js:24:14786)
    at null.ActionsOnFile (/Users/zhangyongkang/RaycastDeveloper/extensions/quick-access/src/search-pinned-directories.tsx:152:11)
    at ray-action-panel
    at Vt (/Applications/Raycast.app/Contents/Resources/RaycastCommands_RaycastCommands.bundle/Contents/Resources/api/node_modules/@raycast/api/index.js:24:17934)
```

Reply:
Mmh, that might be something on our side. Is this only happening for the `OpenWith` action? Can you reproduce it with a minimal example? That would help us debug it a lot quicker.

Reply:
I wrote a test demo and found that when I use `useeffect/usestate` to get the list data source and use `<Action.OpenWith />` to open file/folder in `<List />` view, the warning will occur.

As shown in the code below, if use `testStrings ` as the data source of the list, it will warn; if use `testStrings2,` it will not.

```
import { Action, ActionPanel, List } from "@raycast/api";
import { useEffect, useState } from "react";

export default function Command() {
  const [testStrings, setTestStrings] = useState<string[]>([]);

  useEffect(() => {
    async function _fetchGistContent() {
      const _testStrings: string[] = [];
      for (let i = 0; i < 10; i++) {
        _testStrings.push(`test ${i}`);
      }
      setTestStrings(_testStrings);
    }

    _fetchGistContent().then();
  }, []);

  const testStrings2 = ["test1", "test2", "test3"];

  return (
    <List>
      {testStrings.map((testString, index) => (
        <List.Item
          key={index}
          title={testString}
          actions={
            <ActionPanel>
              <Action.OpenWith path={"/Users/zhangyongkang/RaycastDeveloper/extensions/easy-new-file"} />
            </ActionPanel>
          }
        />
      ))}
    </List>
  );
}
```